dev/conductor/ui/lib/widgets/progression.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:conductor_core/conductor_core.dart';
+import 'package:conductor_core/proto.dart' as pb;
+import 'package:flutter/material.dart';
+
+class MainProgression extends StatefulWidget {
+  const MainProgression({
+    Key? key,
+    required this.releaseState,
+    required this.stateFilePath,
+  }) : super(key: key);
+
+  final pb.ConductorState? releaseState;
+  final String stateFilePath;
+
+  @override
+  MainProgressionState createState() => MainProgressionState();
+}
+
+/// Shows the progression and each step of the release.
+///
+/// 1. ...
```

Comment:
... should mirror what the truth is from conductor

Since they won't be implemented this PR, this is a good time to file GitHub issues against yourself and add them as TODOs in the code